### PR TITLE
Chore/unit test classes

### PR DESCRIPTION
**Pull request by scan-lan**
### What
- Test getter methods in record classes
- Test toString overrides in record classes
- Make App.disconnect static and accept the connection as a parameter
- Styling fixes
- Add teardown to DAO_IntegrationTests 

### Why
- Increase test coverage
- Make testing easier
- Make looking at our code a pleasant experience
- Add coverage to App.disconnect

### CODE REVIEW

File: src/test/java/com/napier/sem/recordClasses_UnitTests.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+package com.napier.sem;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class recordClasses_UnitTests
+{
+    @Test
+    public void capitalCity_ensureGettersReturnsCorrectFields()
```
Comment (scan-lan) [inline src/test/java/com/napier/sem/recordClasses_UnitTests.java]: these names don't mention tostring, which they also test. something to be fixed for tomorrow

File: src/test/java/com/napier/sem/recordClasses_UnitTests.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        assertEquals(name, capitalCity.getName());
+        assertEquals(country, capitalCity.getCountry());
+        assertEquals(region, capitalCity.getRegion());
+        assertEquals(continent, capitalCity.getContinent());
+        assertEquals(population, capitalCity.getPopulation());
+        assertEquals(expectedToStringOutput, capitalCity.toString());
+    }
+
+    @Test
+    public void city_ensureGettersReturnsCorrectFields()
```
Comment (scan-lan) [inline src/test/java/com/napier/sem/recordClasses_UnitTests.java]: ditto

File: src/test/java/com/napier/sem/recordClasses_UnitTests.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        // then
+        assertEquals(name, city.getName());
+        assertEquals(country, city.getCountry());
+        assertEquals(district, city.getDistrict());
+        assertEquals(population, city.getPopulation());
+        assertEquals(expectedToStringOutput, city.toString());
+    }
+
+    @Test
+    public void country_ensureGettersReturnsCorrectFields()
```
Comment (scan-lan) [inline src/test/java/com/napier/sem/recordClasses_UnitTests.java]: ditto

File: src/test/java/com/napier/sem/recordClasses_UnitTests.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
+package com.napier.sem;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
```
Comment (scan-lan) [inline src/test/java/com/napier/sem/recordClasses_UnitTests.java:6]: no docstring -- resolve tomorrow

**Review by LukasP23 — APPROVED** [review]
sick

**Pull request merged**